packages/patternfly-4/react-charts/package.json

```diff
@@ -46,7 +46,7 @@
     "victory-core": "^31.1.0"
   },
   "devDependencies": {
-    "@patternfly/patternfly": "1.0.250",
+    "dg-testrelease": "1.0.10",
```

Comment:
Need to update once core merges to master

Reply:
yup. this was just interim so we could start working on changes.